tm2/pkg/sdk/events.go

```diff
@@ -35,3 +38,43 @@ func (em *EventLogger) EmitEvents(events []Event) {
 // ----------------------------------------------------------------------------
 
 type Event = abci.Event
+
+type DetailedEvent struct {
```

Comment:
This could be called a GnoEvent and it should not be defined in tm2, probably in gnovm, because it is Gno specific.

Reply:
moved all the related things into `gnovm/stdlibs/std/emit_event` file, and also created `Package` file to register them.